<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_settings</class>
 <widget class="QDialog" name="dlg_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>230</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QCheckBox" name="chk_archive">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Archive imported files</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Stores a copy of the imported file in the data folder of the application</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_duplicate_check">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duplicate check during import</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Checks if an expense from an import file already exists in the database, and skips it if it does.</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spn_months_planned">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>12</number>
   </property>
   <property name="value">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of months to add planned expenses in advance</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>10</y>
     <width>20</width>
     <height>211</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>190</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
